plugins/polio/js/src/pages/GroupedCampaigns/GroupedCampaignDialog.tsx

```diff
@@ -91,7 +92,9 @@ export const GroupedCampaignDialog: FunctionComponent<Props> = ({
         },
         [reset],
     );
-    const allowConfirm = Boolean(groupedCampaignName);
+    const allowConfirm =
+        Boolean(groupedCampaignName) && typeof campaignsToLink === 'string';
```

Comment:
`Boolean (campaignsToLink )` is not working ?

Reply:
`Boolean(campaignsToLink)` would indeed be slightly better. We're safe here because the `Select` puts constraints on the value, but `typeof '' === 'string'` is `true` whereas `Boolean("")` is `false`

Reply:
@quang-le @beygorghor The problem with` Boolean(camapignsToLink)` when you open the dialog and fill the name field, the allowConfirm will be set to `true` because campaignsToLink is an empty array and Boolean(campaingsToLInk) will be set to true but we onlt want to be set to true if a campaign is selected and the value is a string.
[Uploading recording-2022-11-14-14-12-46.webm…]()

Reply:
The mistake was initially mine when we shipped the feature. We left a workaround that was never fixed. I'll fix it now. basically, you need to ensure that the type of `campaignsToLink` remains constant, in this case an array